### ISSUE TITLE
Fast charging issues 

### ISSUE THREAD

**Issue by lakshyajoshii** [PixelExperience/android-issues#4488]
## Build date

PixelExperience_miatoll-13.0-20230108-0338-OFFICIAL

## Expected Behavior

The device supports fast charging and i am charging with original charger(33 watt) and usb and it takes roughly around 1 to 1:30 hr to completely charge 

## Current Behavior

It is now taking atleast a hour or 1:30hr to go from 50 to 100%

## Possible Solution

The rom doesn't support fast charging or need some tweaking  

## Steps to Reproduce

1.first of all you have to wait on lock screen until not recognising face
2.then after limited period of face recognisation you have to connect charger plug
3.after that you can see what's happening

/device miatoll
/version thirteen
![Screenshot_20230218-004009_Pixel Launcher.png](https://user-images.githubusercontent.com/85167854/219765514-82c640e6-afb7-49bc-9b9a-0ec5df049a42.png)

**Comment by pixelexperiencebot** [comment]
Issue created! You can close at any time by commenting ```/close```

**Comment by Mustafa7822** [comment]
Same issue is happening with me bro but I am facing the this issue recently pehle nahi hota tha aisa kuch abhi 3 4 dino se horaha hai aur proper backup bhi nahi milraha battery ka

**Comment by Aquaman447** [comment]
Facing The same problem on same device. Problem is True.

**Comment by GuiRibi** [comment]
same here, was plugged in all day and max it reached was 44% and while using the phone the battery level decreased.  (Curtana)
![Screenshot_20230226-201257_Gmail](https://user-images.githubusercontent.com/107240371/221434750-e103f593-81b7-40cd-9ff7-2bbf1cf5ee92.png)

**Comment by dereference23** [comment]
Duplicate of #4353

**Comment by pixelexperiencebot** [comment]
Issue closed by maintainer request.
